src/mono/wasm/debugger/BrowserDebugProxy/EvaluateExpression.cs

```diff
@@ -382,6 +382,13 @@ private static async Task<IList<JObject>> ResolveElementAccess(IEnumerable<Eleme
             return values;
         }
 
+        internal static string RemoveNullSuppression(string expression)
+        {
+            expression = expression.Trim();
+            expression = expression.Replace("!.", ".").Replace("![", "[").Replace("!(", "(");
```

Comment:
don't remove these here because these can be in a quoted string in the expression, like `foo.Bar("x ?.qwe")`, which shouldn't be removed.

Reply:
Good point. Moving the removal it to `ResolveAsInstanceMember`.